src/TreeMenu.js

```diff
@@ -78,6 +78,7 @@ const Menu = ({
      */
```

Comment:
Also this would not be a sufficient fix to the problem as these are just the orphan resources that are getting pushed at the bottom of the tree. We'd need to add the same hasList filter to Line 54-55 where we're pushing the child resources under their hierarchical parents in the tree.